### DISCUSSION TITLE
windowTitle

### DISCUSSION THREAD

**Discussion by ssate** [FelixKratz/SketchyBar#64] [Q&A] [answered]
Hi,

Could you please provide an example of how the windowTitle.sh plugin can be used with the custom events? 

Struggling to figure out how it would actually be implemented in to the bar itself.

Thanks in advance.

**Comment by FelixKratz** [comment, marked as answer]
This will help you to get this running https://github.com/FelixKratz/SketchyBar/discussions/12#discussioncomment-1215932

**Comment by ssate** [comment]
Thanks a lot, got it working! Also, great job on this - the project looks very promising, excited to see the finished product!
